### PR TITLE
handbook: eng > security page

### PR DESCRIPTION
**Pull request by beyang**

### CODE REVIEW

File: handbook/engineering/security.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+All development laptops have encrypted hard drives.
+
+## Ops
+
+Sourcegraph instances that host private code are deployed on-premises and therefore Sourcegraph
+employees have no access to customer data.
+
+On-premises Sourcegraph instances do not send any customer code to other servers. Additional
+customer private data such as email addresses is never sent (with the exception of emails collected
+explicitly for sales and support purposes).
```
Comment (dadlerj) [inline handbook/engineering/security.md]: I'd replace this section with 

`Additionally, other than the email address of the initial installer (for customer support, security, and product notification purposes) Sourcegraph never sends any private user data, such as usernames or email addresses, or other specific data to any other servers. Learn more in our [pings documentation](https://docs.sourcegraph.com/admin/pings). https://docs.sourcegraph.com/admin/pings`

File: handbook/engineering/security.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+On sourcegraph.com, Sourcegraph uses role-based access controls to ensure only administrators have
```
Comment (dadlerj) [inline handbook/engineering/security.md]:
```suggestion
Sourcegraph uses role-based access controls to ensure only customer administrators have access to private configuration data, such as code host and authentication service access tokens.

On sourcegraph.com, Sourcegraph uses role-based access controls to ensure only administrators have
```

File: handbook/engineering/security.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+direct access to production compute resources and sensitive configuration data. All persistent
+volumes are encrypted.
```
Comment (tsenart) [inline handbook/engineering/security.md]: It is news to me that all persistent volumes are encrypted. Can you point me to the relevant Kubernetes manifests?

File: handbook/engineering/security.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Internal traffic and RPCs between services that comprise Sourcegraph are TLS-encrypted.
+
+## Code permissions
+
+Sourcegraph respects code host access controls. Unit and integration tests protect the correctness
+of this permissions check.
```
Comment (tsenart) [inline handbook/engineering/security.md]:
```suggestion
of this permissions checks.
```

**Pull request merged**